
On i.MXL bits per pixel selection in LCD controller's PCR register is slightly
different than on i.MX2+, so mark this difference.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: linux-2.6.36/arch/arm/plat-mxc/include/mach/imxfb.h
===================================================================
--- linux-2.6.36.orig/arch/arm/plat-mxc/include/mach/imxfb.h	2010-11-25 18:27:50.000000000 +0100
+++ linux-2.6.36/arch/arm/plat-mxc/include/mach/imxfb.h	2010-11-25 18:28:45.000000000 +0100
@@ -15,8 +15,12 @@
 #define PCR_BPIX_4	(2 << 25)
 #define PCR_BPIX_8	(3 << 25)
 #define PCR_BPIX_12	(4 << 25)
+#ifdef CONFIG_ARCH_MX1
+#define PCR_BPIX_16	(4 << 25)
+#else
 #define PCR_BPIX_16	(5 << 25)
 #define PCR_BPIX_18	(6 << 25)
+#endif
 #define PCR_PIXPOL	(1 << 24)
 #define PCR_FLMPOL	(1 << 23)
 #define PCR_LPPOL	(1 << 22)
